content/tutorials/move-resources-between-stacks/index.md

```diff
@@ -0,0 +1,397 @@
+---
+title: "Move Resources Between Stacks"
+title_tag: "Move Resources Between Stacks"
+layout: single
+
+# A succinct description of the tutorial. It appears on the Tutorials home and collection pages.
+description: Learn how to move resources between stacks or projects.
+
+# A similar description used for search results and social-media previews.
+meta_desc: Learn how to move resources between stacks or projects.
+
+# An image for the tutorial. It appears on tutorial page and in social-media previews.
+meta_image: move-resources.png
+
+# An optional video for the tutorial. When present, it appears at the top of the page, replacing
+# the meta image. YouTube and HTML5 video sources are supported.
+# video:
+#     url: /blog/drift-detection/drift.mp4
+#     youtube: Q8tw6YTD3ac
+
+# The order in which the tutorial appears in most lists. Order is ascending, so higher numbers
+# mean the tutorial will appear further down the list. Positive integers only.
+weight: 999
+
+# A brief summary of the tutorial. It appears at the top of the tutorial page. Markdown is fine.
+summary: |
+    There are certain scenarios in which you might need to move resources between different project stacks without recreating them, such as when refactoring a Pulumi project from a monolithic structure to micro-stacks. While it is possible to accomplish this by manually modifying Pulumi state files, doing so requires significant effort, can be error prone, and can be very time consuming.
+    
+    In this tutorial, you will learn how to move your resources using the `pulumi state move` command instead.
+
+# A list of three to five things the reader will have learned by the end of the tutorial.
+youll_learn:
+    - How to identify a resource's URN
+    - How to move a resource between stacks
+    - Why and how to use aliases
+
+# A list of tutorial prerequisites. Markdown is fine. Keep it simple; no need to be exhaustive here.
+prereqs:
+    - A [Pulumi Cloud account](https://app.pulumi.com/signup) and [access token](/docs/pulumi-cloud/accounts/#access-tokens)
+    - The [Pulumi CLI](/docs/install/)
+    - An [Amazon Web Services](https://aws.amazon.com/) account
+    - The [AWS CLI](https://aws.amazon.com/cli/)
+    - Your desired [language runtime installed](/docs/clouds/aws/get-started/begin/#install-language-runtime)
+
+# The estimated time, in minutes, for new users to complete the topic.
+estimated_time: 15
+
+# # An optional list of collections this tutorial should be belong to. Collections are defined in data/tutorials/collections.yaml.
+# collections:
+#     - some-non-existent-collection
+---
+
+## Create a new project
+
+To start, login to the [Pulumi CLI](/docs/cli/commands/pulumi_login/) and ensure it is [configured to use your AWS account](/docs/clouds/aws/get-started/begin/#configure-pulumi-to-access-your-aws-account). Next, [create a new project](/docs/clouds/aws/get-started/create-project/) and replace the default program code with the following:
```

Comment:
Once the CLI login tutorial maybe update this link to point to that instead of the `pulumi login` command.